apps/passport-client/components/shared/cards/SecretPhrase.tsx

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
ideally, all this logic should be in CardBody.tsx of the pcd to minimize changes to passport-client

Reply:
moved everything to @pcd/secret-phrase-pcd. I had to hook into `appConfig` in `apps/passportClient/src/pcdPackages.ts` in order to feed it the 3rd party verify URL. If there is a preferred way to do this let me know but adding this seemed like the least overhead/ changes to the client